nservicebus/hosting/nservicebus-host/installation.md

```diff
@@ -20,83 +32,103 @@ NServiceBus.Host.exe [/install [/serviceName]
 [/sideBySide]
 [/startManually]
 [/username]
-[/password]]
-[/uninstall [/serviceName]
-[/sidebyside]
-[/instance:Instance Name ]
+[/password]
 ```
 
-To retrieve this list, run the following at the command line:
+Here is an example of the `/install` command line:
 
 ```dos
-NServiceBus.Host.exe /?
+NServiceBus.Host.exe /install 
+/serviceName:"MyPublisher"
+/displayName:"My Publisher Service"
+/description:"Service for publishing event messages"
+/endpointConfigurationType:"QualifiedNameSpace.EndpointConfigType, AssemblyName"
+/username:"corp\serviceuser"
+/password:"p@ssw0rd!" NServiceBus.Production
 ```
 
-To set the actual name of the Windows Services in the registry, specify `/serviceName:ServiceName`. This is different from what is displayed in the Windows Service Manager.
+
+### ServiceName
+
+To set the actual name of the Windows Services in the registry, specify `/serviceName:ServiceName`. By default, the name of the service is the name of the endpoint and the endpoint name is the namespace of the endpoint configuration class.
```

Comment:
I think the behavior is still as described since the host provides the parameter to the `EndpointConfiguration`

it seems to use:

```
endpointName = specifier.GetType().Namespace ?? specifier.GetType().Assembly.GetName().Name;
var configuration = new EndpointConfiguration(endpointName);
```